etc/encoding/failed.properties

```diff
@@ -0,0 +1,6 @@
+# copy video into mp workspace
+profile.failed.http.name = Failed video
+profile.failed.http.input = visual
+profile.failed.http.output = visual
+profile.failed.http.suffix = -failed.mp4
+profile.failed.http.ffmpeg.command = -i /tmp/failed1.mp4 -c copy #{out.dir}/#{out.name}#{out.suffix}
```

Comment:
Hmm... While I'm not a big fan of having binary files in a GIT repo, this profile obviously will cause the workflow to fail which seems very undesirable.

Can you please somehow make this profile and the workflow that uses it work out-of-the-box?

Also, please add somewhat more description to the profile - it does have a purpose more than "copy video into mp workspace", doesn't it?

Note that you also could use an image and transform it into a video using WOH image-to-video.